src/hal/hal_lib.c

```diff
@@ -2334,6 +2334,7 @@ hal_pin_t *halpr_find_pin_by_name(const char *name)
     int next;
     hal_pin_t *pin;
     hal_oldname_t *oldname;
+    
```

Comment:
this shouldn't be here, I'll try to remember to eliminate this change before merging your pull request